src/base.mli

```diff
@@ -101,3 +101,23 @@ val available : unit -> bool
 [@@@js.implem
 let available () =
   Js_of_ocaml.Js.Optdef.test Js_of_ocaml.Js.Unsafe.global##.mixpanel]
+
+val set_group :
+  group_key:string ->
+  group_ids:string list ->
+  ?callback:(Ojs.t -> unit) ->
+  unit ->
+  unit
+  [@@js.global "mixpanel.set_group"]
+
+module Group : sig
+  type group_id
+
+  (*Pas sûr de si c'est bien des properties*)
+  val get_group : group_key:string -> group_id:Ojs.t -> group_id
```

Comment:
```suggestion
  val get_group : group_key:string -> group_id:string -> group_id
```

Maybe we can simply type the parameter `group_id` this way to catch all [supported data types for properties](https://help.mixpanel.com/hc/en-us/articles/115004547063-Properties-Supported-Data-Types)? What do you think?

Reply:
I hadn't see this this page when I made this function, I only see that the API wanted an "object" as argument, but indeed it's easier to convert all that 5 type into string value instead of an Ojst.t, and I don't think that a "list" should be used as a group_id in application.